tests/e2e/app-files/config.yml

```
@@ -0,0 +1,3 @@
+default:
+  rhino_log_level: !expr Sys.getenv("RHINO_LOG_LEVEL", "TRACE")
+  rhino_log_file: !expr Sys.getenv("RHINO_LOG_FILE", "log.txt")
```

In this case we can simplify the config slightly, as we don't need the ability to change it via env variables here.

```suggestion
  rhino_log_level: TRACE
  rhino_log_file: log.txt
```